### DISCUSSION TITLE
Can this work on top of gnome?

### DISCUSSION THREAD

**Discussion by NullVoxPopuli** [leftwm/leftwm#1041] [Q&A]
I asked a similar question here, but for Wayland: https://github.com/i3-gnome/i3-gnome/issues/115

The reason why I'd like a layer on top of gnome, is that gnome has some really nice features -- and surprisingly, a lot of which are for interfacing with hardware, such as display brightness, battery power, etc.
Gnome also has _really good_ screen capture / recording tool via the printscreen button.

**Comment by VuiMuich** [comment]
It can work on top of a gnome install, but not on top of gnome session, as there is always only one WM allowed to run in a Xorg session.

Generally getting all those nice little features running is considered exactly the adventure part of WMs like leftWM. Depending on the distro you are building your system with, there is varying quality of documentiation on how to achieve things. Arch Linux has tremendous good documentation and therefore is very popular among Linux adventurers for example.
